shared/chat/conversation/info-panel/menu/index.tsx

```diff
@@ -106,13 +106,17 @@ const TeamHeader = (props: TeamHeaderProps) => {
         <Kb.Text type="BodySemibold" style={styles.maybeLongText} onClick={props.onViewTeam}>
           {props.teamname}
         </Kb.Text>
-        <Kb.Meta
-          backgroundColor={Styles.globalColors.blueGrey}
-          color={Styles.globalColors.black_50}
-          icon="iconfont-people-solid"
-          iconColor={Styles.globalColors.black_20}
-          title={teamHumanCount}
-        />
+        {teamHumanCount ? (
```

Comment:
Just confirming: 0 is not a valid real value for `teamHumanCount`, correct?

Reply:
If you're a restricted bot in a team that had just an owner and you , and then the owner reset their account, you could end up with it being 0, I think. But I also don't think we care about that case.